portal-web/tests/js/report.js

```diff
@@ -0,0 +1,194 @@
+(function (window, document) {
+    "use strict";
+
+    var tabs = {};
+
+    function changeElementClass(element, classValue) {
```

Comment:
not related

Reply:
Don't understand why these files have been added, they were under the "test" folder, but I don't know if they were added automatically to the commit because I always made "git add file_name" and only wiht the .testcase, .macro and .path files we are working on.